app.go2tv.go2tv.yml

```diff
@@ -0,0 +1,41 @@
+app-id: app.go2tv.go2tv
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version "24.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    version: '23.08'
```

Comment:
this should be 24.08. It should match the runtime version.

Reply:
ty